database/postgres/pgdatabase.go

```diff
@@ -221,3 +228,20 @@ func quoteColumns(columns []string) []string {
 	}
 	return ret
 }
+
+type execer interface {
```

Comment:
I'd avoid creating an "execer" type here if not absolutely necessary.
Our code base has too many "execers" already IMO.

If I understand correctly, this is only used in deleteResourceByCQId, that receives a "tx". can it just receive a plain "tx"?

Reply:
Fixed.